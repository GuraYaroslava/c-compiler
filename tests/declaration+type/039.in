int a = 10;

char b[a];