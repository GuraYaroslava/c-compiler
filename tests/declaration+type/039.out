[Ln: 3, Col: 10] Error: expression must have a constant value
